src/main/java/it/gov/pagopa/payment/service/payment/common/CommonStatusTransactionServiceImpl.java

```diff
@@ -21,7 +21,7 @@ public CommonStatusTransactionServiceImpl(TransactionInProgressRepository transa
 
     public SyncTrxStatusDTO getStatusTransaction(String transactionId, String merchantId, String acquirerId) {
         TransactionInProgress transactionInProgress= transactionInProgressRepository.findByIdAndMerchantIdAndAcquirerId(transactionId, merchantId, acquirerId)
-                .orElseThrow(() -> new ClientExceptionNoBody(HttpStatus.NOT_FOUND,"Transaction does not exist"));
+                .orElseThrow(() -> new TransactionNotFoundOrExpiredException(ExceptionCode.TRX_NOT_FOUND_OR_EXPIRED,"Transaction does not exist"));
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor